README.md

```diff
@@ -797,6 +797,15 @@ params do
 end
 ```
 
+Alternative (simpler) example using coerce_with in params:
+This will take a string in input and output an Array of Integers.
+Like the Color parser example, coerce_with could have used a class with a parse method.
+```ruby
```

Comment:
Add an empty line above. Backtick-quote the method names, `coerce_with` and `parse`. Basically make it look like the rest of the README (and if you find any other inconsistencies please feel free to fix).